runtime/core/cmake/libtorch.cmake

```diff
@@ -54,4 +64,7 @@ if(TORCH)
     )
   endif()
   add_definitions(-DUSE_TORCH)
+  if(GPU)
+    add_definitions(-DUSE_GPU)
```

Comment:
I think set the cuda version here is better.
BTW, maybe you should print some message when `USE_GPU` in Windows and MacOS.

Reply:
like cmake -DUSE_GPU=11.3 ？
just  print 'not support yet', is it ok?